.github/workflows/cypress-e2e-parallelization.yml

```diff
@@ -339,8 +339,12 @@ jobs:
     runs-on: ubuntu-24.04
     if: |
       inputs.is_nightly == 'true' && github.run_attempt == 1 &&
-      (failure() || cancelled())
+      (failure() || cancelled()) &&
+      startsWith(github.ref_name, 'dev')
```

Comment:
e2e tests will not run in release branches ?

Reply:
the nightly is not triggered on release branches

Reply:
my bad it's only a job to create a ticket